src/main/java/io/cdap/plugin/gcp/bigquery/source/BigQuerySource.java

```diff
@@ -171,6 +172,9 @@ public void prepareRun(BatchSourceContext context) throws Exception {
     if (config.getViewMaterializationDataset() != null) {
       configuration.set(BigQueryConstants.CONFIG_VIEW_MATERIALIZATION_DATASET, config.getViewMaterializationDataset());
     }
+    String temporaryTableName = String.format("%s_%s", config.getTable(),
```

Comment:
minor nit: can you please prefix this by `_`? Tables that start with `_` are not displayed in the BQ Console by default.

Reply:
changed